source/guides/network/running-a-symbol-node-manually.rst

```diff
@@ -0,0 +1,194 @@
+
+The next sections explain first how to build a :ref:`peer-node` and then expand it to convert it into an :ref:`api-node`.
+
+.. note:: The following instructions have only been verified to work on Linux.
+
+*****************
+Build a Peer node
+*****************
+
+Build the server
+================
+
+Follow the `Catapult server <https://github.com/nemtech/catapult-server>`__ instructions to **build** its binaries, either `manually <https://github.com/nemtech/catapult-server/blob/main/docs/BUILD-manual.md>`__ or using `Conan <https://github.com/nemtech/catapult-server/blob/main/docs/BUILD-conan.md>`__.
+
+Run the server
+==============
+
+**Download** the network configuration file from **A LOCATION YET TO BE DETERMINED**.
+
+Follow Catapult server's `instructions to run a node <https://github.com/nemtech/catapult-server/blob/main/docs/RUNPEERLIN.md>`__ using the downloaded network configuration.
+
+*****************
+Build an API node
+*****************
+
+Once you have the Catapult Server running, you can add **extra services** to turn it into an API node:
+
+.. figure:: ../../resources/images/diagrams/rest-detail.png
+    :align: center
+
+As shown in the diagram above, besides the **Catapult Server** you will also need:
+
+- A **Database** holding the REST data.
+- A **Catapult Broker** serializing accesses to the database.
+- A **REST gateway** accepting client requests and turning them into server commands or database queries.
+
+The following sections explain how to install each service. You will need all of them.
+
+Move to the ``_build`` folder created while building the server and create any additional folders requested in the next steps inside ``_build``.
+
+Run the database
+=====================
+
+1. **Install** `MongoDB <http://mongodb.com>`__ (at least version 4.4).
+
+   It is recommended to follow the `program's own installation instructions <https://docs.mongodb.com/manual/administration/install-community/>`__, and NOT install from your distro's packages.
+
+2. Create a directory to store the data files and **run the database**:
+
+   .. code-block:: bash
+
+      mkdir dbfiles
+      mongod --dbpath=dbfiles --wiredTigerCacheSizeGb 2 --bind_ip 127.0.0.1
+
+   Keep ``mongod`` running for as long as your node runs.
+
+3. **Build indices** to optimize database accesses.
+
+   The ``catapult-server`` repository you cloned to build the server in the first step contains `a folder with mongo scripts <https://github.com/nemtech/catapult-server/blob/main/scripts/mongo>`__. Run this one:
+
+   .. code-block:: bash
+
+      mongo 127.0.0.1/catapult < ../../scripts/mongo/mongoDbPrepare.js
```

Comment:
if the current directory is _build, then the command should be:
`mongo 127.0.0.1/catapult < ../scripts/mongo/mongoDbPrepare.js`
`../../` --> `../`

Reply:
👍